include/fluent-bit/flb_log_event_decoder.h

```diff
@@ -81,4 +81,6 @@ int flb_event_decoder_decode_object(struct flb_log_event_decoder *context,
 int flb_log_event_decoder_next(struct flb_log_event_decoder *context,
                                struct flb_log_event *record);
 
+const char* flb_log_event_encoder_strerror(int result);
```

Comment:
Please separate the asterisk from the type name.
Additionally, change `strerror` for `get_error_description`.

This applies both for the encoder and decoder.

Reply:
And rename the argument name to `error_code`

Reply:
Thank you for reviewing. I fixed it.